ReactSkia/sdk/TVNavigationEventEmitter.cpp

```diff
@@ -0,0 +1,40 @@
+#include "TVNavigationEventEmitter.h"
+#include <glog/logging.h>
+#include <iostream>
+#include "RNSKeyCodeMapping.h"
+using namespace std;
+namespace facebook {
+namespace react {
+
+RSkTVNavigationEventEmitter::RSkTVNavigationEventEmitter(
+    const std::string &name, 
+    std::shared_ptr<CallInvoker> jsInvoker,
+    Instance *bridgeInstance)
+    : RSkEventEmitter(name, jsInvoker, bridgeInstance)
+{
+
+}
+void RSkTVNavigationEventEmitter::onTVKeyEvent(int  eventType,int eventKeyAction/*, const unsigned int tag*/) {
+    cout <<"RSkTVNavigationEventEmitter::onTVKeyEvent "<<endl;
+    folly::dynamic parameters = folly::dynamic::object();
+    parameters["eventType"] = RNSKeyMap[eventType];
+    parameters["tag"] = 0xFFFF;
```

Comment:
could you please confirm the default tag value (as per our tests on Android)? 0xFFFF was just set as a default for testing only.

Reply:
Sure.